<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWidget</class>
 <widget class="QWidget" name="StartWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StartWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 16pt &quot;JetBrains Mono&quot;;
</string>
  </property>
  <layout class="QGridLayout" name="backLayout">
   <item row="1" column="0">
    <spacer name="_mainLeftSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="_mainTopSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="_mainRightSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="_mainBottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="mainGroup">
     <property name="styleSheet">
      <string notr="true">QGroupBox#mainGroup{
	font:600 italic 36pt &quot;JetBrains Mono&quot;;
	color:rgb(0, 0, 127);
	border-color:rgb(0, 0, 127);
}

QGroupBox {
	border-width:2px;
	border-style:solid;
	border-radius:10px;
	border-color:gray;
	margin-top:1.6ex;
}

QGroupBox::title{
	subcontrol-origin:margin;
	subcontrol-position:top centorl;
	padding:0 1px;
}
</string>
     </property>
     <property name="title">
      <string>Hex King</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="topMargin">
       <number>24</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="infoLayout_1">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="infoLayout_4">
          <property name="rightMargin">
           <number>16</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="orderLayout">
            <item>
             <widget class="QLabel" name="orderLabel">
              <property name="text">
               <string>Order:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="orderComboBox">
              <property name="currentText">
               <string>5 x 5</string>
              </property>
              <item>
               <property name="text">
                <string>5 x 5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6 x 6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7 x 7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>8 x 8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>9 x 9</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10 x 10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>11 x 11</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>12 x 12</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>13 x 13</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="modeLayout">
            <item>
             <widget class="QLabel" name="modeLabel">
              <property name="text">
               <string>Mode:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="modeComboBox">
              <item>
               <property name="text">
                <string>PvP</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PvE</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>EvE</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Debug</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="attackerGroup">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
	margin-left:2ex;
	margin-right:2ex;
}
</string>
          </property>
          <property name="title">
           <string>Player Attacker</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <property name="spacing">
            <number>9</number>
           </property>
           <property name="leftMargin">
            <number>36</number>
           </property>
           <item>
            <widget class="QRadioButton" name="BlackRadio">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Black</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="WhiteRadio">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>White</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="infoLayout_2">
        <property name="spacing">
         <number>16</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>16</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>16</number>
        </property>
        <item>
         <widget class="QGroupBox" name="blackAIGroup">
          <property name="title">
           <string>Black AI</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>16</number>
           </property>
           <property name="topMargin">
            <number>16</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="b_versionLayout">
             <property name="spacing">
              <number>9</number>
             </property>
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QLabel" name="b_versionLabel">
               <property name="text">
                <string>Version:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="b_versionCombo">
               <item>
                <property name="text">
                 <string>Mcts_A</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Mcts_E</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Mcts_G</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="b_ecfLayout">
             <property name="spacing">
              <number>9</number>
             </property>
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QLabel" name="b_ecflabel">
               <property name="text">
                <string>ECF:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="b_doubleSpin">
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="maximum">
                <double>2.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="b_TimeLayout">
             <property name="spacing">
              <number>9</number>
             </property>
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QLabel" name="b_TimeLabel">
               <property name="text">
                <string>Time:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="b_Time_SpinBox">
               <property name="maximum">
                <number>300</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="b_threadLayout">
             <property name="spacing">
              <number>9</number>
             </property>
             <property name="leftMargin">
              <number>16</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QRadioButton" name="b_ParallelizedRadio">
               <property name="text">
                <string>Parallelized</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="b_SingleRadio">
               <property name="text">
                <string>Single </string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="whiteAIGroup">
          <property name="title">
           <string>White AI</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>16</number>
           </property>
           <property name="topMargin">
            <number>16</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="w_versionLayout">
             <property name="spacing">
              <number>9</number>
             </property>
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QLabel" name="w_versionLabel">
               <property name="text">
                <string>Version:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="w_versionCombo">
               <item>
                <property name="text">
                 <string>Mcts_A</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Mcts_E</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Mcts_G</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="w_ecfLayout">
             <property name="spacing">
              <number>9</number>
             </property>
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QLabel" name="w_ecflabel">
               <property name="text">
                <string>ECF:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="w_doubleSpin">
               <property name="maximum">
                <double>2.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="w_TimeLayout">
             <property name="spacing">
              <number>9</number>
             </property>
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QLabel" name="w_TimeLabel">
               <property name="text">
                <string>Time:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="w_Time_SpinBox">
               <property name="maximum">
                <number>300</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="w_threadLayout">
             <property name="spacing">
              <number>9</number>
             </property>
             <property name="leftMargin">
              <number>16</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QRadioButton" name="w_ParallelizedRadio">
               <property name="text">
                <string>Parallelized</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="w_SingleRadio">
               <property name="text">
                <string>Single </string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="infoLayout_3">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <spacer name="startButtonLeftSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="styleSheet">
           <string notr="true">font: 600 italic 24pt &quot;JetBrains Mono&quot;;</string>
          </property>
          <property name="text">
           <string>Start Game</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="startButtonRightSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
